hole-punch-interop/rust-relay/setRustTarget.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+# Translate Docker platform format to Rust target format
+case "$(echo $1 | cut -d/ -f2)" in
+    "amd64") RUST_TARGET="x86_64-unknown-linux-musl";;
+    "arm64") RUST_TARGET="aarch64-unknown-linux-gnu";;
```

Comment:
Why `-gnu` and not `-musl`?

Reply:
It's a Tier 2 target, not sure if it's a problem. I'll do that.